src/main/java/roomescape/controller/RoomescapeApiController.java

```diff
@@ -0,0 +1,53 @@
+package roomescape.controller;
+
+import jakarta.validation.Valid;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import roomescape.dto.ReservationListResDto;
+import roomescape.dto.ReservationReqDto;
+import roomescape.exception.ErrorCode;
+import roomescape.exception.NotFoundReservationException;
+import roomescape.domain.Reservation;
+import roomescape.service.ReservationService;
+
+import java.util.List;
+
+import static org.springframework.http.HttpStatus.CREATED;
+
+@RestController
+@RequestMapping("/reservations")
+public class RoomescapeApiController {
+    private final ReservationService reservationService;
+    public RoomescapeApiController(ReservationService reservationService) {
+        this.reservationService = reservationService;
+    }
```

Comment:
```suggestion
    private final ReservationService reservationService;
    
    public RoomescapeApiController(ReservationService reservationService) {
        this.reservationService = reservationService;
    }
```
- 변수와 생성자 사이는 한 칸 띄워주시면 더 좋을 것 같아요!